crates/subspace-service/src/lib.rs

```diff
@@ -178,6 +178,10 @@ pub enum Error {
     #[error(transparent)]
     BlockRelay(#[from] BlockRelayConfigurationError),
 
+    /// Snap sync failed.
+    #[error(transparent)]
+    SnapSync(#[from] snap_sync::Error),
```

Comment:
Are you sure this is necessary? I don't think the error will propagate to this level, it'll stop at `if let Err()` below and will not go any further.

Reply:
Thanks for catching that, it was left over from before I split the error types.